src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge18.java

```diff
@@ -0,0 +1,36 @@
+package org.owasp.wrongsecrets.challenges.docker;
+
+
+import org.owasp.wrongsecrets.RuntimeEnvironment;
+import org.owasp.wrongsecrets.ScoreCard;
+import org.owasp.wrongsecrets.challenges.Challenge;
+import org.owasp.wrongsecrets.challenges.Spoiler;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+import static org.owasp.wrongsecrets.RuntimeEnvironment.Environment.DOCKER;
+
+@Component
+@Order(18)
+public class Challenge18 extends Challenge {
+
+    public Challenge18(ScoreCard scoreCard) {
+        super(scoreCard);
+    }
+
+    @Override
+    public Spoiler spoiler() {
+        return new Spoiler("hunter2");
+    }
+
+    @Override
+    public boolean answerCorrect(String answer) {
+        return "hunter2".equals(answer);
```

Comment:
Would it be an idea to compare it to a hash of the input? This is "greppable". Of course that would mean revising the spoiler as well - probably that could work along the lines of challenge 15 (w/ encryption)

Reply:
I compared to the hash in the end, As for the spoiler this wasnt possible so ive ended up using bse64 to obfuscate. I figured this is enough as if the user really want to cheat they can just use the spoiler.

Reply:
This solution looks good to me!